PokeDemo/CustomViews/TableViewCells/PokemonTableViewCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  PokemonTableViewCell.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import UIKit
+
+class PokemonTableViewCell: BaseTableViewCell {
+    
+    // MARK: - Outlets
+    
+    @IBOutlet weak var lblTitle: UILabel!
```

Comment:
can be renamed as title

Reply:
Done in 318a882a3ad1532005f3e532a0215cdf07ec67ed